applications/swbatch/run.sh

```diff
@@ -0,0 +1,79 @@
+# SWbatch v0.2.1
+# Written by: Joseph Vantassel (jvantassel@utexas.edu)
+# Written on: 5 March 2019
+# This code performs batch inversions that consider multiple paramaterizations
+
+# Change log
+# Last edit: 24 June 2019
+# Update to loop over 2 targets, allow for more than 9 seeds, move extraction inside the loop.
+# Last edit: 14 February 2020
+# Bug fix with Np.
+# Last edit: 17 May 2020
+# Major refactoring.
+
+# Setup
+set -x
+WRAPPERDIR=$( cd "$( dirname "$0" )" && pwd )
+cd ${workingDirectory}
```

Comment:
Does this work at the moment? I believe in order for `workingDirectory` to be an env var, you have to use the `envKey` parameter. Otherwise you can reference the directory as a string instead of a variable.

Reply:
I added "envKey": "workingDirectory" to the fileInputs